aws-iam-role-security-audit/module_test.go

```diff
@@ -0,0 +1,45 @@
+package role_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAWSIAMRoleReadOnly(t *testing.T) {
+	t.Parallel()
+
+	region := "us-west-1"
```

Comment:
Yeah I didn't do that because I didn't know where to put shared code, but since go will just treat this like a go project, we can create directory for shared test code.